packages/blade-old/src/molecules/Modal/Modal.native.js

```diff
@@ -118,6 +118,7 @@ const Modal = ({ children, variant, visible, onClose, onBackdropClick }) => {
       <Space margin={[0]}>
         <Flex justifyContent="flex-end">
           <RNModal
+            testID="bottomsheet-modal"
```

Comment:
```suggestion
            testID="ds-bottomsheet-modal"
```
I think we add `ds` as a prefix as a convention to DS components, check once.

Also, how about adding `testID` as a prop & give a default value to it?

Reply:
looks good now, resolving